projects/kit/components/checkbox/checkbox.style.less

```diff
@@ -19,8 +19,7 @@
         height: @sizeL;
     }
 
-    &._disabled,
```

Comment:
Why did you drop `&._disabled` ?

Reply:
`div[tuiWrapper]` has disabled class and `pointer-events: none`